ui/app/AppLayouts/Chat/components/MessageContextMenu.qml

```diff
@@ -130,7 +130,12 @@ PopupMenu {
                   //% "Reply to"
                   qsTrId("reply-to")
         onTriggered: {
-            messageContextMenu.isProfile ? chatsModel.joinChat(fromAuthor, Constants.chatTypeOneToOne) : showReplyArea()
+            if (messageContextMenu.isProfile) {
+                tabBar.currentIndex = 0
```

Comment:
I created a funciton that is more expicit for changing sections:
https://github.com/status-im/nim-status-client/blob/master/ui/app/AppMain.qml#L59

So you just need to call it like this:
```suggestion
                appMain.changeAppSection(Constants.chat)
```